app/presenters/blacklight/rendering/max_length.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Blacklight
```

Comment:
Is this an Override?

Reply:
Nope, new class. Could be namespaced to OregonDigital, but it's completely compatible with BL. Same with other max_values below.